src/TestProblem/Hydro/Zeldovich/Init_TestProb_Zeldovich.cpp

```diff
@@ -0,0 +1,537 @@
+
+
+// (3) reset other general-purpose parameters
+//     --> a helper macro PRINT_WARNING is defined in TestProb.h
+   const long   End_Step_Default = __INT_MAX__;
+   const double End_T_Default    = 1.0;
+
+   if ( END_STEP < 0 ) {
+      END_STEP = End_Step_Default;
+      PRINT_WARNING( "END_STEP", END_STEP, FORMAT_LONG );
+   }
+
+   if ( END_T < 0.0 ) {
+      END_T = End_T_Default;
+      PRINT_WARNING( "END_T", END_T, FORMAT_REAL );
+   }
+
+#  ifdef PARTICLE
+   if ( Gas_Par_Setup == 1 ) // overwrite the total number of particles in gas-only setup
+   {
+      amr->Par->NPar_Active_AllRank = 0;
+   }
+   if ( Gas_Par_Setup == 2 ) // overwrite the total number of particles in particle-only setup
+   {
+      amr->Par->NPar_Active_AllRank = NPar_X*SQR(NPar_YZ);
+   }
+#  endif
+
+
+// (4) make a note
+   if ( MPI_Rank == 0 )
+   {
+      Aux_Message( stdout, "=============================================================================\n" );
+      Aux_Message( stdout, "  test problem ID                   = %d\n",     TESTPROB_ID     );
+      Aux_Message( stdout, "  gas[1]/particle[2] setup          = %d\n",     Gas_Par_Setup   );
+      Aux_Message( stdout, "  perturbation wavelength multiple  = %14.7e\n", n_Pert_Wave_Len );
+      Aux_Message( stdout, "  Zeldovich collapse redshift       = %14.7e\n", zc_Collapse     );
+      Aux_Message( stdout, "  initial gas temperature in Kelvin = %14.7e\n", GasTemp_Init    );
+      Aux_Message( stdout, "  particle number along the x-axis  = %d\n",     NPar_X          );
+      Aux_Message( stdout, "=============================================================================\n" );
+   }
+
+
+   if ( MPI_Rank == 0 )    Aux_Message( stdout, "   Setting runtime parameters ... done\n" );
+
+} // FUNCTION : SetParameter
+
+
+
+// compute conversion between Lagrangian (x_Lagrangian) and Eulerian/code coordinates (x_shift)
+double Zeldovich_coord_transformation(double x_Lagrangian_Trial, void *params)
+{
+   struct Zeldovich_coord_params *p = (struct Zeldovich_coord_params *) params;
+   double x_shift_input = p->x_shift_input;
+   double zi_Initial = p->zi_Initial;
+
+   return (x_Lagrangian_Trial - ((1+zc_Collapse)/(1+zi_Initial))*sin(k_Pert*x_Lagrangian_Trial)/k_Pert) - x_shift_input;
+}
+#ifdef SUPPORT_GSL
+   double Zeldovich_coord_transformation_solver(double x_shift_input, double zi_Initial)
+   {
+      int status;
+      int iter = 0, max_iter = 100;
+      const gsl_root_fsolver_type *T;
+      gsl_root_fsolver *s;
+      double x_Lagrangian_Trial = x_shift_input;
+      double x_lower_bound = x_shift_input-Pert_Wave_Len/4, x_upper_bound = x_shift_input+Pert_Wave_Len/4;
+      gsl_function F;
+      struct Zeldovich_coord_params params = {x_shift_input, zi_Initial};
+
+      F.function = &Zeldovich_coord_transformation;
+      F.params = &params;
+
+      T = gsl_root_fsolver_brent;
+      s = gsl_root_fsolver_alloc (T);
+      gsl_root_fsolver_set (s, &F, x_lower_bound, x_upper_bound);
```

Comment:
Remove the space before `(`. Add a space after `(` and before `)`. Check the entire file.